packages/code-du-travail-frontend/src/contributions/__tests__/extractMdxContentUrl.test.js

```diff
@@ -0,0 +1,13 @@
+import extractMdxContentUrl from "../extractMdxContentUrl";
+
+const tests = [
+  '<Content href="https://www.service-public.fr/professionnels-entreprises/vosdroits/F33693"/>',
```

Comment:
I would add some label for test cases to improve readability, like
```
[
{ 
label: "Content Only", 
data:  "'<Content href="https://www.service-public.fr/professionnels-entreprises/vosdroits/F33693"/>", 
url: "https://www.service-public.fr/professionnels-entreprises/vosdroits/F33693"
}, 
{ 
label: "Start with Content": 
data: "..."
url: ""
}]
```
that way you get assert on matchedUrl and get rid of snapshots

Reply:
OK

Reply:
Done